### PR TITLE
Added BTRFS backup and recovery docs.

### PR DESCRIPTION
**Pull request by adamradocz**

### CODE REVIEW

File: docs/backup_(btrfs,crashplan).md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+#### What and Why?
+There are many technologies in the storage space and the ease of use, cost, and reliability vary greatly. Here we'll attempt to articulate why these particular technologies were used versus others available.
+
+##### BTRFS
+---
+###### What?
+Btrfs is a modern copy on write (CoW) filesystem for Linux aimed at implementing advanced features while also focusing on fault tolerance, repair and easy administration.
+###### Why?
+BTRFS will be used for pooling and also for the filesystem. The data modell will be RAID5, but BTRFS has the flexbility to switch between data models if needed eg. RAID1 to RAID5/6.
```
Comment (trapexit) [inline docs/backup_(btrfs,crashplan).md]: typo: modell

File: docs/backup_(btrfs,crashplan).md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+* **Why not EXT4/XFS?**
```
Comment (trapexit) [inline docs/backup_(btrfs,crashplan).md]: Might want to expand this. The logical volume management, snapshotting, bitrot detection, etc.

**Comment by adamradocz** [comment]
I modified the file. This is what you meant about expanding the Why not EXT4/XFS section?

File: docs/backup_(btrfs,crashplan).md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+#### 3. Add to /etc/fstab
+Use the UUID of the newly created BTRFS.
+```
+# <file system>                           <mount point>  <type>  <options>  <dump>  <pass>
+UUID=3a6b5f69-23bc-4698-a945-d06062aa5ff6 /mnt/storage   btrfs    defaults    0       2
+
+```
+
+#### 4. Prepare CrashPlan
+Follow the [CrashPlan setup guide](setup_(crashplan).md). Create a "backup set" for each type of data you wish to backup (phtoos, music, etc.).
```
Comment (trapexit) [inline docs/backup_(btrfs,crashplan).md]: typo: phtoos

Comment (adamradocz) [reply docs/backup_(btrfs,crashplan).md]: I just copied the text from the backup_(mergerfs,crashplan).md, so there must be a type there too. :)
I'll fix it.

**Pull request merged**